exercises/concept/bird-count/.meta/Exemplar.elm

```diff
@@ -0,0 +1,58 @@
+module BirdCount exposing (busyDays, hasDayWithoutBirds, incrementDayCount, today, total)
+
+
+today : List Int -> Maybe Int
+today counts =
+    case counts of
```

Comment:
I would use List.head here

Reply:
Right, but all these are pretty much one liners if you use `List`, except for `incrementDayCount`.

```elm
today = List.head
hasDayWithoutBirds = List.any ((==) 0)
total = List.sum
busyDays = List.filter ((>=) 5) >> List.length
```